CenterForOpenScience/ember-osf-web
# [EMB-561] Fix Registries.discover to recognize test.osf.io registrations

## Purpose

Fix Registries.discover to recognize test.osf.io registrations

## Summary of Changes

- Fix regex used to determine the main link for registrations of OSF source
- Add tests

## Side Effects

N/A

## Feature Flags

N/A

## QA Notes

Check and make sure registration created on test.osf.io appears in search results with test.osf.io links.

## Ticket

[EMB-561](https://openscience.atlassian.net/browse/EMB-561)

# Reviewer Checklist
